src/main/proto/th2_grpc_act_uiframework_win_demo/uiframework_win_demo.proto

```diff
@@ -91,11 +91,5 @@ message ActResponse {
   ExecutionStatus scriptStatus = 2;
   string statusInfo = 3;
   string errorInfo = 4;
-  map<string, ResponseData> data = 5;
-  string executionId = 6;
-
-  message ResponseData {
-    string value = 1;
-    MessageType messageType = 2;
-  }
+  map<string, string> data = 5;
```

Comment:
Why we change API ?

Reply:
We revert to previous version of API for compatibility with current version of act-gui-core and th2-act-uiframework-win-demo. These changes were not yet merged into master branch ( https://github.com/th2-net/th2-act-gui-core/pull/36 and https://github.com/th2-net/th2-act-uiframework-win-demo/pull/3 ).